File: TripKitAndroidUI/src/main/java/com/skedgo/tripkit/ui/geocoding/ReviewSummary.java
package com.skedgo.tripkit.ui.geocoding;

import com.google.gson.annotations.SerializedName;

public class ReviewSummary {

    @SerializedName("averageRating")
    public float averageRating;

    @SerializedName("reviewCount")
    public int reviewCount;

    @SerializedName("ratingImageURL")
    public String ratingImageURL;
}

File: TripKitAndroidUI/src/main/java/com/skedgo/tripkit/ui/geocoding/ReviewSummary.kt
package com.skedgo.tripkit.ui.geocoding

import com.google.gson.annotations.SerializedName

class ReviewSummary {
    @SerializedName("averageRating")
    var averageRating: Float = 0f

    @SerializedName("reviewCount")
    var reviewCount: Int = 0

    @SerializedName("ratingImageURL")
    var ratingImageURL: String? = null
}